doc/dev-guide/src/tracing.md

```diff
@@ -101,7 +101,7 @@ when enabled. Instrumenting a currently uninstrumented function is mostly simply
 done like so:
 
 ```rust
-#[tracing::instrument(level = "trace", err, skip_all)]
+#[tracing::instrument(level = "trace", err(level = "trace"), skip_all)]
```

Comment:
What does this mean/do?

Reply:
This is for disabling error tracking for these functions unless the level is set to "TRACE". The default level is "DEBUG" (see https://docs.rs/tracing/latest/tracing/attr.instrument.html#examples-2) which is way too high, and will interfere with the default logger: https://github.com/rust-lang/rustup/actions/runs/9867247214/job/27247322849

One thing to note is that the current "legacy" logging formatter has ignored spans altogether, this might be worth fixing as well, but probably in another PR. The point is running `rustup` with `RUSTUP_LOG=rustup=TRACE` emits the right spans and their associated errors (when they do occur).